include/igl/copyleft/marching_cubes.cpp

```diff
@@ -25,38 +25,40 @@
 
 #include "marching_cubes.h"
 #include "marching_cubes_tables.h"
-#include <map>
+
+#include <unordered_map>
```

Comment:
I think the map --> unordered_map change will be easier to merge. Is it easy for you to put in a separate PR?